package.json

```diff
@@ -34,7 +34,7 @@
     "stage": "NODE_ENV=production npm run production && gcloud app deploy --project=web-dev-staging --quiet",
     "start": "node ./server",
     "test:watch": "ava --watch",
-    "pretest": "cd tests/e2e/fixtures && eleventy",
+    "pretest": "cd tests && npx rollup -c && cd e2e/fixtures && eleventy",
```

Comment:
@matthiasrohmer WDYT? This means we run both eleventy and rollup before e2e tests as well as unit tests. should we split it apart?

Reply:
With the new puppeteer testes introduced here I think that makes sense.